docs/_static/usage.inc

```diff
@@ -12,5 +12,5 @@ The following is a basic example of using the **nidmm** module to open a session
         session.configureMeasurementDigits(nidmm.Function.DC_VOLTS, 10, 5.5)
         print("Measurement: " + str(session.read()))
 
-Additional examples for each driver are located in src/<driver>/examples/ directory.
+Additional examples for each driver are located in src/<driver>/examples/ directory `on GitHub <https://github.com/ni/nimi-python/tree/master/src/<driver>/examples>`_.
```

Comment:
This link will not work (does this render as a link?)

Reply:
Does it need to work? The existing path is bogus already. What's the purpose of this file?

Reply:
Is the file dead code? If so kill it 💀 !
It might have been used in the past.